lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -199,7 +199,7 @@ class IndexQueryTestCase(ModuleStoreTestCase):
     NUM_PROBLEMS = 20
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 8),
+        (ModuleStoreEnum.Type.mongo, 9),
```

Comment:
Yes, but notice that the count didn't go up for Split Modulestore.  For Split, since the sequence module is rendered within an outer "bulk-operation", the course is cached in the local thread and not refetched from Mongo.  

I'm not worried about the Old Mongo Modulestore, as we are moving away from using it.